ccmlib/node.py

```diff
@@ -889,7 +889,18 @@ def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_out
                     p.stdin.write(cmd + ';\n')
             p.stdin.write("quit;\n")
 
-            output = p.communicate(timeout=timeout)
+            try:
+                output = p.communicate(timeout=timeout)
+            except subprocess.TimeoutExpired as e:
+                self.error(f"(EE) {e}")
+                pat = re.compile(r'\r*\n')
```

Comment:
not sure why we need this regex for, wouldn't we print the whole outout/stdout ?